src/lib/configs.ts

```diff
@@ -20,7 +22,20 @@ export async function initializeConfigs(config: Config = {}): Promise<{ [key: st
   GLEE_DIR = path.resolve(GLEE_PROJECT_DIR, '.glee')
   GLEE_LIFECYCLE_DIR = path.resolve(GLEE_DIR, config.functionsDir || 'lifecycle')
   GLEE_FUNCTIONS_DIR = path.resolve(GLEE_DIR, config.functionsDir || 'functions')
-  GLEE_CONFIG_FILE_PATH = path.resolve(GLEE_DIR, 'glee.config.js')
+
+  GLEE_CONFIG_FILE_PATH_TS = path.resolve(GLEE_DIR, 'glee.config.ts')
+  GLEE_CONFIG_FILE_PATH_JS = path.resolve(GLEE_DIR, 'glee.config.js')
+  GLEE_CONFIG_FILE_PATH = existsSync(GLEE_CONFIG_FILE_PATH_TS) ? GLEE_CONFIG_FILE_PATH_TS : GLEE_CONFIG_FILE_PATH_JS
+
+  if(existsSync(GLEE_CONFIG_FILE_PATH_TS) && existsSync(GLEE_CONFIG_FILE_PATH_JS)) {
```

Comment:
```suggestion
  const configJSExists = existsSync(GLEE_CONFIG_FILE_PATH_JS)
  const configTSExists = existsSync(GLEE_CONFIG_FILE_PATH_TS)
  GLEE_CONFIG_FILE_PATH = configTSExists ? GLEE_CONFIG_FILE_PATH_TS : GLEE_CONFIG_FILE_PATH_JS

  if(configTSExists && configJSExists) {
```

Consider minimizing the times we access the disk. That would speed up the process and would make it less error-prone.